docs/operator-manual/applicationset/Generators-SCM-Provider.md

```diff
@@ -410,6 +410,7 @@ spec:
 * `pathsDoNotExist`: An array of paths within the repository that must not exist. Can be a file or directory.
 * `labelMatch`: A regexp matched against repository labels. If any label matches, the repository is included.
 * `branchMatch`: A regexp matched against branch names.
+* `includeArchivedRepos`: Include repositories that are archived.
```

Comment:
To be in the main config, it should be supported and implemented in all scm providers. I think this should be moved to the specific github and gitea if they are the only scm supporting it.

Reply:
Sounds good, I'll make that change.